### ISSUE TITLE
Facets: Write HOWTO for changing: Removing, adding if the data already exists.

### ISSUE THREAD

**Issue by robredpath** [ThreeSixtyGiving/grantnav#567]
- It's clear how the world will be different when the work is completed

Someone who has a broad understanding of GN, elastic and Python will be able to add or remove facets, because there will be a helpful step-by-step guide to where the relevant code is, how it works, and a worked example of a change. 

- It's clear what will and won't be done

This will probably be a 2-step job, with someone setting out the basics during other facet work (eg #568), and someone writing it up. These may or may not be the same person. 

- Any uncertainty, assumptions, dependencies or constraints are recorded

#571 changes how this would be done, so we should wait before doing this. 

- There's an order-of-magnitude estimate of how long it will take

Hours

- The work's been approved by someone at ThreeSixtyGiving

Yes

- There are links to everything that's written down about the work - such as plan.io tickets, GitHub issues

**Comment by michaelwood** [comment]
@kindly Do we have anything more to do on this ticket?

**Comment by kindly** [comment]
I do not think we got round to actually writing the HOWTO above the miro workflow we did.

Howerver, I made the process of adding/removing a terms facet much easier.  i.e just need to add line in TermFacets list and copy the template of another facet.

https://github.com/ThreeSixtyGiving/grantnav/blob/current-sprint/grantnav/frontend/views.py#L37

That current-sprint branch needs to be merged too at some point.
